modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -7,20 +7,23 @@
 = Using Data Mover for CSI snapshots
 
 :FeatureName: Data Mover for CSI snapshots
-include::snippets/technology-preview.adoc[]
 
-The OADP 1.1.2 Data Mover enables customers to back up container storage interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs. The OADP 1.1.2 Data Mover solution uses the Restic option of VolSync.
+The Data Mover enables customers to back up Container Storage Interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs.
+
+VolumeSnapshotBackups (VSBs) and VolumeSnapshotRestores (VSRs) are queued using the VolumeSnapshotMover (VSM). The VSM's performance is improved, by specifying a concurrent number of VSBs and VSRs simultaneously in Progress.
+
+After all async plugin operations are completed, the backup is marked as completed.
+The Data Mover solution uses the Restic option of VolSync.
+
 
 [NOTE]
 =====
 Data Mover supports backup and restore of CSI volume snapshots only.
-
-Currently, Data Mover does not support Google Cloud Storage (GCS) buckets.
 =====
 
 .Prerequisites
```

Comment:
@eemcmullan @CarmiWisemon maybe we should add this line under Prerequisites
Verify oadp namespace has below annotation. 
`oc annotate --overwrite namespace/openshift-adp volsync.backube/privileged-movers='true'`

Reply:
@PrasadJoshi12 +1 for volsync > 0.7

Reply:
This annotation was added in volsync 0.6 for privileged movers. Not sure if it is worth specifying in cases of someone using an older version of volsync. Or we can include that OADP 1.2 only supports volsync > 0.7 (current version). 

@weshayutin wdyt?